glide.yaml

```diff
@@ -353,6 +353,8 @@ import:
   version: 72f9bd7c4e0c2a40055ab3d0f09654f730cce982
 - package: github.com/juju/ratelimit
   version: 5b9ff866471762aa2ab2dced63c9fb6f53921342
+- package: github.com/json-iterator/go
+  version: f2b4162afba35581b6d4a50d3b8f34e33c144682
```

Comment:
same as before, separate these out into another commit.

Reply:
In order to quickly address I made the minimum change here which was to carve out change to glide.yaml from bump into it's own commit "pinned depedndencies"